### DISCUSSION TITLE
Warning: No available formula with the name "gcc" (dependency of coin-or-tools/coinor/cbc).

### DISCUSSION THREAD

**Discussion by mosa11am** [Homebrew/discussions#2518] [Getting started]
I wanted to download Cbc solver for python. I updated my macbook to download Xcode. I downloaded the latest version of homebrew and followed the instructions in the following link.

https://github.com/coin-or/Cbc

brew tap coin-or-tools/coinor
brew install coin-or-tools/coinor/cbc

however i got stuck here and received that error

Warning: No available formula with the name "gcc" (dependency of coin-or-tools/coinor/cbc)

**Comment by SMillerDev** [comment]
Did you try asking them about it? I don't really know how coinor runs their homebrew tap.

**Reply by mosa11am** [reply]
not yet. but I will. Thanks!

**Comment by gromgit** [comment]
If your Homebrew setup can't find a key formula like `gcc`, it's likely that your setup is messed up.

What's the output of `brew config` and `brew doctor`?

**Reply by RolandBarro** [reply]
already on master
